static/news.en.html

```diff
@@ -1,12 +1,29 @@
 <h1 id="helpPage_newsTitle">News</h1>
 
+<h2 class="title">A new game mode, choice of complexity and more</h2>
+<span class="date">
+    __.08.2020
+</span><span class="version">
+    v1.2
+</span><span class="new">
+    * New
+</span>
+<p>
+The game has become more comfortable and interesting!
+
+<li>A new game mode with words from players: every participant writes words from which the set of words forms</li>
+<li>The Russian dictionaries with different complexity appeared</li>
+<li>If you have your own dictionary, you can upload it and play with words from it</li>
+<li>You can now limit not the number of words but the number of rounds</li>
+<li>The number of turns for your next participation is now displaying</li>
+<li>After the end of the game a common link for next game appears</li>
```

Comment:
Пока нет перевода кнопки "играть ещё раз", могу ненароком накосячить.
```suggestion
<li>After the end of the game a common link for next game appears. Press "play one more time" to play again all together</li>
```

Reply:
Может быть, кнопка "играть ещё раз" будет "play again"? И тогда. наверное, лучше сделать так "Press "play again" to play one more time all together"

Reply:
Я перевёл так, как перевёл, потому, что:
1. "One more time" и "again" соотносятся так же, как "ещё раз" и "снова". Чисто формальный перевод "ещё раз" – "one more time", а "again" – хороший, но не совсем точный перевод.
1. Я использовал оба перевода, чтобы не было повторения, как его нет в оригинале. В принципе эти выражения можно поменять местами.

Так что да, можно.